### PR TITLE
Fix websocket on old versions of IE11

### PR DESCRIPTION
**Pull request by jwilander**
#### Summary
Should fix the issue with IE11.

@asaadmahmood can you try it on the spinmint with your dev console open? I was only able to reproduce by forcing a websocket disconnect. It seems to work fine on my IE11.

### CODE REVIEW

**Comment by asaadmahmood** [comment]
Nops @jwilander, this still doesn't fix it. If I delete a message, nothing happens, it's updated if I refresh or send another message.
`I get the console message websocket must have connection url`.

**Comment by jwilander** [comment]
Alright... let me add some debugging

**Comment by jwilander** [comment]
Ready for review. Fix confirmed working by @asaadmahmood

File: webapp/actions/websocket_actions.jsx [hunk 1]
```diff
@@ -30,34 +30,37 @@ import {browserHistory} from 'react-router/es6';
 const MAX_WEBSOCKET_FAILS = 7;
 
 export function initialize() {
-    if (window.WebSocket) {
-        let connUrl = Utils.getSiteURL();
+    if (!window.WebSocket) {
+        console.log('Browser does not support websocket'); //eslint-disable-line no-console
```
Comment (enahum) [inline webapp/actions/websocket_actions.jsx]: Maybe we should show this error in the `ErrorBar` or the error page? 0/5

**Pull request merged**